angular-client/src/pages/charging-page/components/high-low-cell/high-low-cell-display/high-low-cell-display.component.ts

```diff
@@ -14,6 +14,13 @@ export default class HighLowCellDisplay {
   highCellVoltage: number = 0;
   mobileThreshold = 1000;
   isDesktop = window.innerWidth > this.mobileThreshold;
+  resetGraph: boolean = false;
```

Comment:
same comment as that other component, you should store the arrays of lowVoltsData and highVoltsData in this component and pass them down as inputs to the graph to replace the resetGraph attribute